kaniko/kaniko.yaml

```diff
@@ -9,13 +9,17 @@ spec:
   - name: DOCKERFILE
     description: Path to the Dockerfile to build.
     default: /workspace/Dockerfile
+  - name: CONTEXT
+    description: Build Context
+    default: /workspace
 
   steps:
   - name: build-and-push
     image: gcr.io/kaniko-project/executor
```

Comment:
I also suggest you add a tag to the image, so that we pin to a specific version of the image instead of going with the latest. I used v0.9.0 for tektoncd pipeline repo.

Reply:
sound good; i'll add the pinned version into the next push